sdk/python/tests/test/BasicMessageEncoderTest.py

```diff
@@ -0,0 +1,34 @@
+from symbolchain.CryptoTypes import PrivateKey
+
+
+class BasicMessageEncoderTest:
+	# pylint: disable=no-member
+
+	def test_sender_can_decode_encoded_message(self):
+		# Arrange:
+		key_pair = self.KeyPair(PrivateKey.random())
+		recipient_public_key = self.KeyPair(PrivateKey.random()).public_key
+		encoder = self.MessageEncoder(key_pair)
```

Comment:
don't you need something like get_message_encoder (abstract) for pylint ?

Reply:
pylint was not complaining, probably due to `pylint: disable=no-member` above (but we do that in all "Basic" tests)

Reply:
i'd probably use abstract for consistency with others but no strong opinion